src/io/flutter/preview/PreviewView.java

```diff
@@ -545,13 +556,8 @@ public void customizeCellRenderer(
       setIcon(icon);
     }
 
-    // Render the parent/child association.
-    if (outline.getParentAssociationLabel() != null) {
-      appendSearch(outline.getParentAssociationLabel() + ": ", SimpleTextAttributes.REGULAR_ATTRIBUTES);
```

Comment:
If we don't show it, we might also exclude it from `getSpeedSearchString`.

Reply:
makes sense; done